java/src/jmri/jmrit/jython/InputWindow.java

```diff
@@ -179,7 +179,10 @@ protected boolean loadFile(JFileChooser fileChooser) {
                     languages.setSelectedItem(JmriScriptEngineManager.getDefault().getFactoryByExtension(Files.getFileExtension(file.getName())).getLanguageName());
                 } catch (ScriptException npe) {
                     log.error("Unable to identify script language for {}, assuming its Python.", file);
-                    languages.setSelectedItem(JmriScriptEngineManager.getDefault().getFactory(JmriScriptEngineManager.PYTHON).getLanguageName());
```

Comment:
Please use a SuppressWarning annotation here—if someone has removed parts of a JMRI installation that we can’t compile without, we should crash on the NPE (or whatever error the JmriScriptEngineManager will throw because missing dependencies are not present instead of logging here.

Reply:
Thanks, changed to annotation.